_posts/0000-01-08-dorinalora.md

```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
```

Comment:
Maybe change the title of your slide, so it doesn't say "Welcome to our slide deck!"